docs/guide/concepts/bots.md

```diff
@@ -38,39 +40,37 @@ You can immediately talk to the bot by calling its `say()` method, which is an a
 await ford_bot.say("Hello!")
 ```
 
-A synchronous convenience method is also available: 
-```python
-ford_bot.say_sync("Hello again!")
-``` 
-
 ### History
-When you speak with a bot, every message is automatically stored. The bot uses its `history` module to access these messages, which means you can refer to earlier parts of your conversation without any extra work. In Marvin, each conversation is called a `thread`. Bots generate a new thread any time they are instantiated, but you can resume a specific thread by calling `Bot.set_thread()`. If you want to clear the thread and start a new one, call `Bot.reset_thread()`. 
-### Saving bots
+When you speak with a bot, every message is automatically stored. The bot uses its `history` module to access these messages, which means you can refer to earlier parts of your conversation without any extra work. 
 
-Bots can be saved to the database by calling the `Bot.save()` method. Bots are saved by name. You can use the `if_exists` argument to customize what happens if a bot with the same name already exists:
+In Marvin, each conversation is called a `thread`. Bots generate a new thread each time they are instantiated.  
```

Comment:
this is not strictly true, bots can use InMemoryHistory in which no threads are created